gobblin-runtime/src/test/java/gobblin/runtime/api/TestJobExecutionState.java

```diff
@@ -44,7 +47,10 @@
 
   @Test public void testStateTransitionsSuccess() throws TimeoutException, InterruptedException {
     final Logger log = LoggerFactory.getLogger(getClass().getSimpleName() + ".testStateTransitionsSuccess");
-    JobSpec js1 = JobSpec.builder("gobblin:/testStateTransitionsSuccess/job1").build();
+    JobSpec js1 = JobSpec.builder("gobblin:/testStateTransitionsSuccess/job1")
```

Comment:
I think requiring a job name in the test is okay since we check for it in other places, but want to double check with @ibuenros.